### DISCUSSION TITLE
Vulnerability Scanner Whitelist

### DISCUSSION THREAD

**Discussion by baileyphil** [Security-Onion-Solutions/securityonion#2895] [General]
Afternoon All, 

Apologies if this has been covered before but i did a search here and in the documentation and couldnt find anything for the new v2.

We have a Vulnerabiilty scanner that has scheduled scans running, i wanted to know if it was possible to whitelist this in SO so it doesnt start giving me false positives?

Thanks 

Phil

**Comment by cm-ops** [comment]
You could use a BPF for the vulnerability scanner as described here https://docs.securityonion.net/en/latest/bpf.html?highlight=BPF.
